frontend/src/pages/projects/screens/spawner/deploymentSize/ContainerSizeSelector.tsx

```diff
@@ -35,7 +35,11 @@ const ContainerSizeSelector: React.FC<ContainerSizeSelectorProps> = ({
         {sizes.map((size) => {
           const { name } = size;
           const desc = getSizeDescription(size);
-          return <SelectOption key={name} value={name} description={desc} />;
+          return name === 'Custom' ? (
```

Comment:
You should avoid disjointed magic strings; always use a constant when you have to share a value across files -- it makes it difficult to maintain, especially with common words like `Custom`.